schemas/com.snowplowanalytics.badrows/storage_loader_failure/jsonschema/1-0-0

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a storage loader failure bad row",
+  "self": {
+    "vendor": "com.snowplowanalytics.badrows",
+    "name": "storage_loader_failure",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "failure": {
+      "type": "object",
+      "description": "Information regarding the storage loader failure",
+      "properties": {
+        "schema": {
+          "type": "string",
+          "description": "Schema of the particular storage loader failure",
+          "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+        },
+        "data": {
+          "type": "object",
+          "description": "Data associated with the particular storage loader failure"
+        }
+      },
+      "required": [ "schema", "data" ],
+      "additionalProperties": false
+    },
+    "payload": {
+      "type": "object",
+      "description": "The collector payload that resulted in a storage loader failure",
+      "properties": {
+        "enrichedEvent": {
+          "type": "string",
+          "description": "Enriched event associated with this failure"
+        },
+        "id": {
+          "type": "string",
+          "description": "Machine-readable unique identifier of the storage loader that produced this failure",
+          "format": "uuid"
+        }
```

Comment:
should this rather belong in the failure or in the processor?

Reply:
@BenFradet to check with @chuwy

Reply:
Depends on our decision regarding "totally invalid data" problem I mentioned above.